vmngclient/api/config_device_inventory_api.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from vmngclient.session import vManageSession
+
+from vmngclient.endpoints.configuration_device_inventory import (
+    ConfigurationDeviceInventory,
+    DeviceUnlockPayload,
+    UnlockDeviceDetail,
+)
+
+
+class ConfigurationDeviceInventoryAPI:
+    def __init__(self, session: vManageSession):
+        self.session = session
+        self.endpoint = ConfigurationDeviceInventory(session)
+
+    def unlock(self, device_uuid: str, device_type: str, device_details: list):
```

Comment:
missing return type

Reply:
Good catch. I'll fix this.

Reply:
Resolved in 46b3022edb376c2cc033c6fd1c27024f74d16fae